atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -374,6 +330,34 @@ def update_repos_from_composes(self):
             self.log.info('adding repo file from compose: %s', compose_info['result_repofile'])
             self.repos.append(compose_info['result_repofile'])
 
+    def _add_filesystem_to_dockerfile(self, file_path, build_dir: BuildDir):
+        """
+        Put an ADD instruction into the Dockerfile (to include the filesystem
+        into the container image to be built)
+        """
+        content = 'ADD {0} /'.format(file_path)
+        lines = build_dir.dockerfile.lines
+
+        custom_image_from = 'FROM koji/image-build'
+        scratch_from = 'FROM scratch'
+
+        for index, line in enumerate(lines):
+            if f'{custom_image_from}\n' not in line and f'{custom_image_from} ' not in line:
+                continue
```

Comment:
I would prefer if we use the `base_image_is_custom()` util method to do this check. IIRC, the image doesn't have to be exactly `koji/image-build`, but can be `koji/image-build:some-tag`.

The util method doesn't work on entire lines of course, but if you work with `parent_images` instead, it should work nicely

Reply:
updated.